include/wil/stl.h

```diff
@@ -187,6 +187,10 @@ namespace wil
 
     using zstring_view = basic_zstring_view<char>;
     using zwstring_view = basic_zstring_view<wchar_t>;
+
+    constexpr zwstring_view operator "" _zw(const wchar_t* str, std::size_t len) noexcept {

```

Comment:
@dmachaj  what marker name do you recommend ? Is there any convention that i should follow ?

Reply:
i am updating it with zv. (similar to sv (string view))

Reply:
Looks like for user defined literal _ is necessary.

build log:
D:\a\1\s\include\wil/stl.h(191,28): error: user-defined literal suffixes not starting with '_' are reserved; no literal will invoke this operator [-Werror,-Wuser-defined-literals]
    constexpr zstring_view operator "" zv(const char* str, std::size_t len) noexcept {
                           ^
D:\a\1\s\include\wil/stl.h(195,29): error: user-defined literal suffixes not starting with '_' are reserved; no literal will invoke this operator [-Werror,-Wuser-defined-literals]
    constexpr zwstring_view operator "" zv(const wchar_t* str, std::size_t len) noexcept {
                            ^

Reply:
That's a bummer but it makes sense for the standard library to reserve names.  `_zv` seems reasonable.

I'd also like us to consider `_zsv` as well.  That would be similar to how `zwstring_view` is like `wstring_view` with a z on the front.  So the literal would be `sv` with a z on the front to make `zsv`.  (And then add the underscore because it is required).

I sent mail to a few other reviewers to ask for additional opinions on this topic.

Reply:
the email concluded at using "_zv".